HW_8/HW_8.3_Divisible_numbers.py

```diff
@@ -0,0 +1,26 @@
+"""
+3. Find all of the numbers from 1-1000 that are divisible by 7
+"""
+
+# Defining list variable with elements from range 1 to 1000
+numbers_list = list(range(1, 1000))
+
+# Filtering input sequence using comprehension
+processed_list = [number for number in range(1, 1000) if number % 7 == 0]
+
+
+def divisible_number(sequence: list) -> list:
+    """
+    Function for processing and filtering list based on specific condition: divisible by 7
+    """
+    result_list = []
+    for each in sequence:
+        if each % 7 == 0:
+            result_list.append(each)
+    return result_list
```

Comment:
it is a task for comprehension. Function it too much for this task

Reply:
Sure. It was intended as double-check and to show that comprehension performs the same things with less amount of code.